public/refs/components/schemas/StudyArm.json

```diff
@@ -1,39 +1,21 @@
 {
-  "description": "An account",
+  "description": "The study arm",
   "type": "object",
   "properties": {
-    "id": {
-      "type": "number",
-      "description": "The id of the study arm",
-      "example": 1
-    },
     "display_name": {
       "type": "string",
       "description": "The display name of the study arm",
-      "example": "LoremB6H7E"
+      "example": "Post-Treatment Monitoring Arm"
     },
-    "created_at": {
```

Comment:
Are we removing created_at because it's no longer getting used?

Reply:
yup, it was never used